packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -225,10 +226,10 @@ describe('Call Tests', () => {
     expect(call).toBeTruthy();
     /* After creation , call manager should have 1 record */
     expect(Object.keys(callManager.getActiveCalls()).length).toBe(1);
-    call.mute(localAudioStream);
+    call.mute(localAudioStream, MUTE_TYPE.USER);
```

Comment:
These tests should not require changes and must still pass.

Reply:
That's correct, removed them now